spec/system/candidate_interface/course_selection/candidate_signed_in_user_with_course_params_spec.rb

```diff
@@ -4,6 +4,7 @@
   include CourseOptionHelpers
   scenario 'candidate is signed in' do
     given_the_pilot_is_open
+    and_apply_again_with_three_choices_are_inactive
```

Comment:
It looks like there are 4 system specs that have this flag set to inactive. This is good. But should each of them have a corresponding spec with the feature flag active? (I can only see one new one and that may be fine if there is duplication there).

Other suggestions would be about optimising for removal of the feature flag in future:
1. copy the original tests with a `_three_choices_feature_inactive` filename suffix. 
2. Deactivate the feature flag in the copies and mark them as 'to be deleted when removing the feature flag'. 
3. Activate the feature flag in the originals and make whatever changes are needed there.
That way you only need to delete the redundant specs when dropping the feature flag at some time in the future.

Reply:
Thanks, that totally makes sense and will definitely make deleting the code easier. 

I've done this for the tests which won't be redundant when we delete the flag and added a few comments. Let me know what you think?